packages/@react-spectrum/tooltip/docs/Tooltip.mdx

```diff
@@ -0,0 +1,175 @@
+import Edit from '@spectrum-icons/workflow/Edit';
+import {Flex} from '@react-spectrum/layout';
+import Question from '@spectrum-icons/workflow/Question';
+import Resize from '@spectrum-icons/workflow/Resize';
+import Save from '@spectrum-icons/workflow/SaveTo';
+import {Text} from '@react-spectrum/text';
+import ThumbUp from '@spectrum-icons/workflow/ThumbUp';
+import { Tooltip, TooltipTrigger } from '@react-spectrum/tooltip';
+```
+
+---
+category: Overlays
+keywords: [tooltip]
+---
+
+# Tooltip
+
+<p>{docs.exports.Tooltip.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Tooltip']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/tooltip/'}
+  ]} />
+
+## Examples
+Try hovering or focusing the button. On hover, it'll appear after a short delay. On focus it'll appear instantly. Clicking or hitting Escape will dismiss it.
+```tsx example
+<TooltipTrigger>
+  <ActionButton aria-label="Edit Name"><Edit /></ActionButton>
+  <Tooltip>Change Name</Tooltip>
+</TooltipTrigger>
+```
+
+### Delay
+The above mentioned delay after hover can be adjusted. Focus will always appear instantly. Setting it to 0 will cause hover to be instant as well.
+```tsx example
+<TooltipTrigger delay={0}>
+  <ActionButton aria-label="Save"><Save /></ActionButton>
+  <Tooltip>Saving applies your new settings right away.</Tooltip>
+</TooltipTrigger>
+```
+
+## Content
+
+Tooltips can have a label provided as a child. This can be an internationalized string.
+
+### Accessibility
+
+Tooltips will use their content to describe their trigger, they should not be complex. Tooltip triggers must be focusable and hoverable in order to ensure that all users can activate them.
+
+## Tooltip placement
+
+Tooltips support a variety of placement options since they do not take over the user interface.
+
+### Placement
+
+The popover's placement with respect to its anchor element can be adjusted using the `placement`
+prop. See the props table [below](#tooltiptrigger-props) for a full list of available placement combinations.
+
+```tsx example
+<TooltipTrigger placement="end">
+  <ActionButton aria-label="Foo">Trigger</ActionButton>
+  <Tooltip>Something</Tooltip>
+</TooltipTrigger>
+```
+
+### Offset and cross offset
+
+The Tooltip's offset with respect to its trigger can be adjusted using the `offset` and
+`crossOffset` props. The `offset` prop controls the spacing applied along the main axis between the element and its
+trigger whereas the `crossOffset` prop handles the spacing applied along the cross axis.
+
+Below is a tooltip offset by an additional 50px above the trigger.
+```tsx example
+<TooltipTrigger offset={50}>
+  <ActionButton aria-label="Offset from trigger">Offset</ActionButton>
+  <Tooltip>This will shift up.</Tooltip>
+</TooltipTrigger>
+```
+Below is a tooltip cross offset by an additional 100px to the right of the trigger.
+```tsx example
+<TooltipTrigger crossOffset={100} placement="bottom">
```

Comment:
maybe make it a little bigger than 100 to really see the shift?

Reply:
so actually, I'm having trouble with this one, it doesn't seem to shift over, i'm trying to decide if i should write a bug or try to take care of it here